build.gradle

```diff
@@ -211,7 +211,7 @@ project(':suro-s3') {
         compile project(':suro-localfile')
 
         compile 'net.java.dev.jets3t:jets3t:0.9.2'
-        compile 'com.amazonaws:aws-java-sdk:1.4.7'
+        compile 'com.amazonaws:aws-java-sdk-s3:1.9.3'
```

Comment:
not sure if this is gonna to be an issue. platform/atlas pulls in the uber jar of aws-java-sdk.